diff --git a/src/ColorRGB.php b/src/ColorRGB.php
index 73a265a..78673cf 100644
--- a/src/ColorRGB.php
+++ b/src/ColorRGB.php
@@ -66,7 +66,7 @@ class ColorRGB extends ColorBase {
    * @return int
    *   The red value
    */
   public function getRed() {
-    return (0.5 + $this->red) | 0;
+    return (int)(0.5 + $this->red) | 0;
   }
 
@@ -76,7 +76,7 @@ class ColorRGB extends ColorBase {
    * @return int
    *   The green value
    */
   public function getGreen() {
-    return (0.5 + $this->green) | 0;
+    return (int)(0.5 + $this->green) | 0;
   }
 
@@ -86,7 +86,7 @@ class ColorRGB extends ColorBase {
    * @return int
    *   The blue value
    */
   public function getBlue() {
-    return (0.5 + $this->blue) | 0;
+    return (int)(0.5 + $this->blue) | 0;
   }
 